### PR TITLE
bug 1490727: Recurring payments thank you page styling.

### PR DESCRIPTION
**Pull request by joshJarr**
Updates the styles for the thank-you page, you can access that from `/payments/success`.

**In this PR**
- https://trello.com/c/U1vtd5hc/29-as-a-user-i-want-to-receive-confirmation-online-that-my-payment-is-successful-so-that-i-can-be-sure-my-payment-has-worked

**Notes for QA**

- [ ] Ensure confirmation page text is readable and makes sense
- [ ] Email links (if displayed) should be clickable and open your native email client
- [ ] Page should be responsive
- [ ] Share links should work as intended
- [ ] The banner popover and on `/payments` should not be affected.

**Screenshots**
screenshot 2018-11-01 at 14 47 47
screenshot 2018-11-01 at 14 47 33
screenshot 2018-11-01 at 14 47 18

### CODE REVIEW

File: kuma/payments/jinja2/payments/thank_you.html [hunk 1]
```diff
@@ -71,8 +74,61 @@ <h2 class="highlight highlight-spanned">
...
+            <section class="section">
+                <div class="index">
+                    <h2>{{_('Useful Things')}}</h2>
+                </div>
+                <div class="faqs clear feedback">
+                    <div class="faq-feedback">
+                        <div class="sub-section">
+                            <h3>{{_('Managing your payments')}}</h3>
+                            <p>
+                                {% trans url="mailto:mdn-support@mozilla.com?Subject=Manage%20monthly%20payment" %}
```
Comment (jwhitlock) [inline kuma/payments/jinja2/payments/thank_you.html]: This should use the setting instead:
```suggestion
                                {% trans url="mailto:" + settings.CONTRIBUTION_SUPPORT_EMAIL + ?Subject=Manage%20monthly%20payment",
                                         email=settings.CONTRIBUTION_SUPPORT_EMAIL  %}
```

Maybe it is a good time to change the name of the setting to ``PAYMENTS_SUPPORT_EMAIL`` as well,

File: kuma/payments/jinja2/payments/thank_you.html [hunk 2]
```diff
@@ -71,8 +74,61 @@ <h2 class="highlight highlight-spanned">
...
+                                    If you would like to manage your monthly payment,
+                                    such as changing your card account details or the
+                                    amount you pay, please contact
+                                    <a href="{{ url }}">mdn-support@mozilla.com</a>.
```
Comment (jwhitlock) [inline kuma/payments/jinja2/payments/thank_you.html]: and then this changes to

```suggestion
                                    <a href="{{ url }}">{{ email }}</a>.
```

File: kuma/payments/jinja2/payments/thank_you.html [hunk 3]
```diff
@@ -71,8 +74,61 @@ <h2 class="highlight highlight-spanned">
...
+                                {% endtrans %}
+                            </p>
+                        </div>
+                        <div class="sub-section">
+                            <h3>{{_('Cancelling your payments')}}</h3>
+                            <p>
+                                {% trans url="mailto:mdn-support@mozilla.com?Subject=Cancel%20monthly%20payment" %}
```
Comment (jwhitlock) [inline kuma/payments/jinja2/payments/thank_you.html]: A similar change is needed here to use the setting.

File: kuma/static/styles/components/payments/banner.scss [hunk 4]
```diff
@@ -188,6 +188,30 @@
             margin-bottom: 1em;
         }
     }
+    &.payment-confirmation {
```
[the remark this answers is not in the document]
Comment (joshJarr) [reply kuma/static/styles/components/payments/banner.scss]: 👍

File: kuma/payments/jinja2/payments/thank_you.html [hunk 5]
```diff
@@ -71,8 +74,61 @@ <h2 class="highlight highlight-spanned">
 
 {% block content %}
     <main id="contributions-page" class="contributions-page">
+        <!-- TODO: replace with waffle flag/varible -->
+        {% if False %}
```
[the remark this answers is not in the document]
Comment (joshJarr) [reply kuma/payments/jinja2/payments/thank_you.html]: Yes, also thinking we need to only show information about managing payment if the user has come from the recurring payments flow, so we'll add a new view to pass that context to this page. 
```
{% if recurring_payment_enabled and user_has_submitted_recurring_payment %}
```

**Comment by joshJarr** [comment]
Unsure why Travis has borked here?

File: kuma/payments/jinja2/payments/thank_you.html (hunk 5 above)
[the remark this answers is not in the document]
Comment (joshJarr) [reply kuma/payments/jinja2/payments/thank_you.html]: This is now done!

**Comment by joshJarr** [comment]
@jwhitlock addressed your feedback here and have tested locally, all seems to be working as intended so far!

File: kuma/payments/jinja2/payments/thank_you.html [hunk 6]
```diff
@@ -71,8 +74,62 @@ <h2 class="highlight highlight-spanned">
...
+                                {% endtrans %}
+                            </p>
+                        </div>
+                    </div>
+                </div>
+            </section>
+        {% endif %}
         <section class="section">
-            <img src="{{ static('img/hero-dino-blank.png') }}" class="backdrop-image" alt="" role="presentation" />
+            <!-- TODO: replace with waffle flag/varible -->
```
Comment (jwhitlock) [inline kuma/payments/jinja2/payments/thank_you.html]: Nit: If you use Jinja2 comments, they don't appear in the rendered HTML

```suggestion
            {# TODO: replace with waffle flag/variable #}
```

File: kuma/payments/views.py [hunk 7]
```diff
@@ -64,8 +64,11 @@ def contribute(request):
 
 @skip_if_disabled
 @never_cache
-def confirmation(request, status):
-    context = {'status': status}
+def confirmation(request, status, recurring=False):
```
Comment (jwhitlock) [inline kuma/payments/views.py]: Nice! This will work to adjust the pages for recurring vs non-recurring.

**Pull request merged**